[IMP] accounting: belgium localization updates (#18106)
odoo/documentation: content/applications/finance/fiscal_localizations/belgium.rst

     - Technical name
     - Description
   * - :guilabel:`Belgium - Accounting`
     - `l10n_be`
     - Belgian :ref:`fiscal localization package <fiscal_localizations/packages>`, complete with
       the Belgian chart of accounts, taxes, tax report, and fiscal positions
   * - :guilabel:`Belgium - Accounting Reports`
     - `l10n_be_reports`
     - Module providing Belgian accounting reports
   * - :guilabel:`Belgium - Accounting Reports (post wizard)`
     - `l10n_be_reports_post_wizard`
     - Enables the VAT wizard when posting a tax return journal entry
   * - :guilabel:`Belgium - Accounting Reports - Prorata Deduction`
     - `l10n_be_reports_prorata`
     - Provides the option to add the prorata deduction to the VAT export
   * - :guilabel:`Belgium - Accounting Reports - SMS`
     - `l10n_be_report_sms`
     - Bridge module between Belgian accounting and SMS
   * - :guilabel:`Belgium - Import SODA files`
     - `l10n_be_soda`
     - Module to import SODA files
   * - :guilabel:`Belgium - Import Bank CODA Statements`
     - `l10n_be_coda`
     - Module to import CODA bank statements
   * - :guilabel:`Belgium - Disallowed Expenses Data`
     - `l10n_be_disallowed_expenses`

>      - Disallowed Expenses Data for Belgium

Reviewer: I think you could shorten it to:

```suggestion
     - Disallowed expenses
```

But again, I'm thinking that this might not be installed automatically ...

Author: I just checked on another runbot in 18.0 and the module seems to be installed automatically. Great suggestion to shorten it, however, I would name it "Disallowed expenses data" to differentiate it from disallowed expenses (fleet)

<img width="390" height="171" alt="image" src="https://github.com/user-attachments/assets/74c133cf-842e-40bd-9479-b8b6d6a68862" />

Taxes
=====

.. important::

>    When submitting your tax returns, ensure to use XML or VAT format files. These are the only file

Reviewer: ```suggestion
   When submitting your tax returns, make sure to use XML or VAT format files. These are the only file
```

Not sure if there's a strict grammatical rule about this (and maybe it's a US vs British/Irish English), but I would only really say 'ensure that ...', and not 'ensure to ...'.

   .. image:: belgium/deductible-tax.png
      :alt: Example of not-fully deductible tax

Vehicle tax deduction
---------------------

.. note::

>    To see the tax deductibility on a vehicle, the **Belgium - Disallowed Expenses Fleet** module

Reviewer: ```suggestion
   To see the tax deductibility of a vehicle, the **Belgium - Disallowed Expenses Fleet** module
```

- 'of' rather than 'on'?
- Also, I notice we also tend to give the technical name of localization modules ... should we add that here? I know it's long, but ...

Author: I added the technical name for the module and also added the list of installed-by-default modules of the localization at the top, to bring it in line with the rest of the localizations :)

Reviewer: Nice!! I didn't dare suggest/ask about that since in general I'm trying to only focus on the updates in a PR review.

Question though: is the Belgium - Disallowed Expenses Fleet one installed automatically? I'm pretty sure I had to install it separately when I created a Belgian company in runbots/a trial DB.

In the Vehicle tax deduction section I think I'd just format the technical name differently ... will add my suggestion.

Author: > Nice!! I didn't dare suggest/ask about that since in general I'm trying to only focus on the updates in a PR review.
> 
> Question though: is the Belgium - Disallowed Expenses Fleet one installed automatically? I'm pretty sure I had to install it separately when I created a Belgian company in runbots/a trial DB.
> 
> In the Vehicle tax deduction section I think I'd just format the technical name differently ... will add my suggestion.

Indeed, the Disallowed Expenses Fleet module has to be installed separately and isn't part of the core packages installed. That's why it's not part of the list of modules at the top and has a note here. Should I make any changes regarding the module? :)

   must be :doc:`installed <../../general/apps_modules>`.

A vehicle's tax deductibility rate varies depending on its type (car or bicycle) and several
factors, such as fuel type, CO2 emissions, engine power, etc.

To view the **tax deductibility percentage** for a specific vehicle, open the **Fleet** app,
navigate to :menuselection:`Configuration --> Models`, and select a vehicle model. Locate the
:guilabel:`Tax Deduction` field, which is found under the :guilabel:`Engine` section for cars, or

> the :guilabel:`Fiscality` section for bicycles.

Reviewer: ```suggestion
To view the **tax deductibility percentage** for a specific vehicle, open the **Fleet** app,
navigate to :menuselection:`Configuration --> Models`, and select a vehicle model. Locate the
:guilabel:`Tax Deduction` field, which is found under the :guilabel:`Engine` section for cars, or
the :guilabel:`Vehicle Information` section for bicycles.
```
I tried this in two runbots and saw this in both:

<img width="419" height="652" alt="Image" src="https://github.com/user-attachments/assets/1ab8350a-f623-4e6f-880d-36940225976f" />

.. important::
   The :guilabel:`Tax Deduction` field is strictly **informative** and is computed automatically
   based on the vehicle's specifications. It is **not** used for any automated calculations within
   the **Accounting** app and should not be confused with the :ref:`tax rate deductibility

>    <belgium/non-deductible>` used on tax grids. Instead, your accountant can reference this field to

Reviewer: ```suggestion
   the **Accounting** app and should not be confused with the :ref:`tax rate deductibility
   <belgium/non-deductible>` used on :ref:`tax grids <tax-returns/tax-grids>`. Instead, your accountant can reference this field to
```

Probably good to link to the doc on 'Tax grids'? Would 👆 be the correct section to link to?

   manually apply the correct deductible rate to invoices or disallowed expenses.

.. seealso::
   - :doc:`Taxes <../accounting/taxes>`
   - :doc:`../accounting/reporting/tax_returns`

>    - :doc:`Fleet: Models and Manufacturers <../../hr/fleet/models>`

Reviewer: Is your goal to give them easy access to more info about vehicle models? Then maybe just:

```suggestion
   - :ref:`Vehicle models <fleet/models>`
```

   .. image:: belgium/deductible-tax.png
      :alt: Example of not-fully deductible tax

Vehicle tax deduction
---------------------

> .. note::
>    To see the tax deductibility of a vehicle, the **Belgium - Disallowed Expenses Fleet
>    (l10n_be_account_disallowed_expenses_fleet)** module must be :doc:`installed
>    <../../general/apps_modules>`.

Reviewer: Maybe like this? 

```suggestion
.. note::
   To see the tax deductibility of a vehicle, the **Belgium - Disallowed Expenses Fleet**
   (`l10n_be_account_disallowed_expenses_fleet`) module must be :doc:`installed
   <../../general/apps_modules>`.
```

A vehicle's tax deductibility rate varies depending on its type (car or bicycle) and several
factors, such as fuel type, CO2 emissions, engine power, etc.

To view the **tax deductibility percentage** for a specific vehicle, open the **Fleet** app,
navigate to :menuselection:`Configuration --> Models`, and select a vehicle model. Locate the
:guilabel:`Tax Deduction` field, which is found under the :guilabel:`Engine` section for cars, or
the :guilabel:`Vehicle Information` section for bicycles.

.. important::
   The :guilabel:`Tax Deduction` field is strictly **informative** and is computed automatically
   based on the vehicle's specifications. It is **not** used for any automated calculations within
   the **Accounting** app and should not be confused with the :ref:`tax rate deductibility
   <belgium/non-deductible>` used on :ref:`tax grids <tax-returns/tax-grids>`. Instead, your
   accountant can reference this field to manually apply the correct deductible rate to invoices or
   disallowed expenses.

.. seealso::
   - :doc:`Taxes <../accounting/taxes>`
   - :doc:`../accounting/reporting/tax_returns`

>    - :ref:`Vehicle Models <fleet/models>`

Reviewer: ```suggestion
   - :ref:`Vehicle models <fleet/models>`
```
